cloudformation/travis.template

```diff
@@ -75,6 +111,13 @@
                     "SecretAccessKey"
                 ]
             }
+        },
+        "TestKMSKey": {
+            "Value": {
+                "Fn::GetAtt": [
+                    "MapboxTileCopyKMSKey"
```

Comment:
@rclark since we have an alias, do I need to export anything and add to the travis UI? If so, I'm not able to find what the specific "resource attribute" is to export.

Reply:
Nope!